components/ProjectTile.jsx

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import { FaExternalLinkSquareAlt, FaGithub } from "react-icons/fa";
+
+const ICON_SIZE = 30;
+
+/* eslint-disable @next/next/no-img-element */
+const ProjectTile = ({ logo, name, liveProjectLink, githubLink, descriptions }) => {
+  return (
+    <div className="flex flex-col items-center bg-white rounded-3xl border-4 border-[#A689FF] p-10 m-10 space-y-4">
```

Comment:
Is it just me or does the Figma file have a bit of a shadow behind each tile?
![image](https://user-images.githubusercontent.com/41935461/185816355-0b2d90c5-4988-4375-895d-02a8f5ea3497.png)